README.md

```diff
@@ -5,7 +5,23 @@ auto-entities
 
 Automatically populate lovelace cards with entities matching certain criteria.
 
-For installation instructions [see this guide](https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins).
+## Installation
+
+### HACS
+
+Search for for `auto-entities` in Plugins. After installing restart Home Assistant.
```

Comment:
"After installing restart Home Assistant." That is not needed.

Reply:
Yes it is, at least in my case (docker based HA installation).